packages/utilities/psammead-assets/src/svgs/plainIcons.jsx

```diff
@@ -0,0 +1,23 @@
+/* Plain Icons contain svgs which are b64 transformed and so cannot depend on any classes
+ */
+import React from 'react';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+const defaultAttrs = {
+  viewBox: '0 0 32 32',
+  width: GEL_SPACING_DBL,
```

Comment:
is `GEL_SPACING_DBL` = 32? If so then should the viewbox use `GEL_SPACING_DBL`?

Reply:
No, `GEL_SPACING_QUAD` is 32. I'll update that